training/tests/provision/playbook.yml

```diff
@@ -56,11 +80,14 @@
     poll: 0
     register: install_result
     ansible.builtin.shell: |
-      podman run --authfile=/etc/containers/auth.json \
-        --rm --privileged --pid=host \ 
```

Comment:
@lmilbaum I saw this previously ran with `--rm` so I did not remove it, but it does not make sense to me why we want this. The instance will already clean itself up, this seems like just introducing another liability with respect to the `bootc paradigm`